proto/cosmos/gov/v1/gov.proto

```diff
@@ -303,4 +303,16 @@ message Params {
   //
   // Since: cosmos-sdk 0.50
   string min_deposit_ratio = 16 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // optimistic_authorized_addresses is an optional governance parameter that limits the authorized accounts than can
+  // submit optimistic proposals
+  //
+  // Since: x/gov v1.0.0
+  repeated string optimistic_authorized_addresses = 17 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
we should avoid storing bech32 encoded address, can we do bytes here?

Reply:
We can, we do need to update client/v2 to support the bech32 conversion from bytes to string then.
Or duplicate params for the query where the query handler does the conversion.

Reply:
Actually updating the params will be annoying as well, so we would need to duplicate `v1.Params` for the query and for state, where the only difference would be the address encoding. Should we really do it?

Reply:
if we do the former is fine. Ideally we dont use the same types for storage and client everywhere but this is another thing we have to roll back. Go with the design you feel most comfortable with

Reply:
What's the former here?

Reply:
I think splitting the type is good. I'll let the proto one be the client and maybe just use a struct for the storage.

Should I do that here or follow/up?

Reply:
I understood: https://github.com/cosmos/cosmos-sdk/pull/18620/commits/14292ae035b42ce5dbbf92ccab48a158cfea34c8, I'll look at client/v2 in the following days, or will create an issue tomorrow.

Reply:
reverted (https://github.com/cosmos/cosmos-sdk/commit/b20186bc528c5de0e6b612b141473305d99e5738) as discussed on slack